lib/routes/hket/index.js

```diff
@@ -106,7 +103,7 @@ module.exports = async (ctx) => {
 
                     // fix lazyload image
                     content('img').each((_, e) => {
-                        content(e).after(`<img src="${content(e).attr('data-src')}" alt="${content(e).attr('data-alt')}">`);
+                        content(e).after(`<figure><img src="${content(e).attr('data-src')}" alt="${content(e).attr('data-alt')}"><figcaption>${content(e).attr('data-alt')}</figcaption></figure>`);
```

Comment:
使用其他容器 wrap 的原因是？建议替换属性，用 cheerio 创建元素

Reply:
> 使用其他容器 wrap 的原因是？

Add captions to images.
In some news articles, there is an image gallery. If we only list the images in the gallery without any description, it would be a bad reading experience. By adding captions to each image, users can have a better understanding of the image. Hence, a better UX.

> 建议替换属性，用 cheerio 创建元素

Fixed in https://github.com/DIYgod/RSSHub/pull/8132/commits/07e0d68a86a245b3d69ce613572d1ec60e810c89